spec/bundler_inject_spec.rb

```diff
@@ -0,0 +1,210 @@
+RSpec.describe Bundler::Inject do
+  shared_examples_for "overrides" do
+    it "with local overrides" do
+      write_bundler_d_file <<~F
+        gem "rack-obama"
```

Comment:
thanks Obama!!!  I totally forgot they used this gem name in their tests.

Reply:
I wanted gems with no dependencies and that don't ever upgrade for simplicity